conduct.md

```diff
@@ -52,35 +52,35 @@ TODO: Write an introductory paragraph.
 
 ### Communication
 
-TODO: Write an introductory paragraph.
+Communication is key to both successful and sustainable code. However it's important that all communication between students and team members follows the three pillars of Respect, Trust, and Humility. In general, students will *respect* their peers, *trust* their contributions and thoughts, and practice *humility* in all aspects of communication, in person, on GitHub, in the Discord, and otherwise.
```

Comment:
There was some discussion in issue #1 about the use of contractions, such as "it's". The latest comment indicated that one group was not using contractions. Is this something that a consensus among all teams has been reached about?

Reply:
To my knowledge we hadn't even talked about it, so I'll just remove the contraction because I think it's more professional.